docs/sources/tutorials/use-angular.md

```diff
@@ -0,0 +1,89 @@
+## Use with Angular application
+
+
+Create file `faro-initializer.ts` and add your SDK configuration :
+
+```typescript
+import { initializeFaro } from '@grafana/faro-web-sdk';
+
+export function faroInitializer(): Function {
+  return async () => {
+      initializeFaro({
+          url: 'https://collector-host:12345/collect',
+          apiKey: 'secret',
+          app: {
+              name: 'frontend',
+              version: '1.0.0',
+          },
+          instrumentations: [...getWebInstrumentations({ captureConsole: true, captureConsoleDisabledLevels: [] })]
+      });
+  };
+}
+```
+
+In the `app.module.ts` init faroInitializer function :
```

Comment:
Let's use not an abbreviation and teh full word instead.
Remove white space before colon.

```suggestion
In the `app.module.ts` initialize the `faroInitializer` function:
```